gulpfile.js

```diff
@@ -150,5 +160,5 @@ function log(error) {
 }
 
 
-gulp.task('default', ['dist', 'copy']);
+gulp.task('default', ['dist', 'copy', 'deploy']);
```

Comment:
@bhomnick from what I understand, dependencies are only guaranteed to work as expected if they return a stream: https://github.com/gulpjs/gulp/blob/master/docs/API.md#deps

It might not matter in this case, but that was my first impression from my experience with gulp.